packages/amplication-server/src/core/git/git.resolver.ts

```diff
@@ -33,7 +33,7 @@ export class GitResolver {
   async createGitRepository(
     @Args() args: CreateGitRepositoryArgs
   ): Promise<Resource> {
-    return this.gitService.createGitRepository(args.data);
+    return this.gitService.createRemoteGitRepository(args.data);
```

Comment:
why did you decide to change the name?

Reply:
To match our convention of calling remote for the non DB git repositories